### PR TITLE
How to build and run the application

### PR DESCRIPTION
**Pull request by ctolon22**

### CODE REVIEW

File: description.md [hunk 1]
```diff
@@ -117,4 +124,5 @@ As we can see our project follows Mavens directory layout. There are some direct
 - [Deploying in Heroku](https://devcenter.heroku.com/articles/deploying-gradle-apps-on-heroku)
 - [Travis CI overview](https://en.wikipedia.org/wiki/Travis_CI)
 - [How to install gradle on windows 7](https://docs.gradle.org/current/userguide/userguide_single.html)
-- [Introduction to the Standard Directory Layout with Maven](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html)
\ No newline at end of file
+- [Introduction to the Standard Directory Layout with Maven](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html)
+>>>>>>> upstream/master
```
Comment (RecuencoJones) [inline description.md]: remove this line

File: description.md [hunk 2]
```diff
@@ -109,6 +109,13 @@ In our case, our project has the next structure:
 
 As we can see our project follows Mavens directory layout. There are some directories in the layout that aren't present in our project structure but this is because our project source doesn't use anything from those directories since they are generated at building time.
 
+##The next instructions will help you to build and run the application:
+
+The first step is to download the project or clone with git (`git clone https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race`). Now, you can see the code and this information. This code is built with gradle. Gradle is a build automation system which uses configuration files like build.gradle which is a build configuration script. To build the code you have to use gradle: `gradle build` on your terminal. After a few seconds, gradle respond you with "BUILD SUCCESSFUL" that means the build has completed.
```
Comment (RecuencoJones) [inline description.md]: this is duplicated, there is [another section](https://github.com/ctolon22/lab1-git-race/blob/master/description.md#how-to-run-the-application-on-the-local-machine) explaining this, please try to improve it

**Comment by ctolon22** [comment]
I have changed it. I have added a new section "How to build the code in the local machine" and i have omitted information related to "run the application" because is in the section "How to run the application  in the local machine"

**Comment by SAReyes** [comment]
I find this info kind of meaningless, I believe most of us can get a quick look at [gradle's userguide](https://docs.gradle.org/current/userguide/userguide) to find this. I'll let other reviewers decide on this. 

I do get the main purpose of this project is to learn about a project lifecycle, but it bugs me when someone adds documentation for third party components without even referencing it out.

File: description.md [hunk 3]
```diff
@@ -47,22 +47,26 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
...
     - Ultimately, edit the variable *PATH* adding *;%JAVA_HOME%\bin* at the end.
 
+
 # HOW TO RUN THE APPLICATION ON THE LOCAL MACHINE
 
-It's easy to run the application on the local machine. You just have to follow the wiki of the main repository to configure the tools needed. After configuiring Gradle you can try to compile using the command `gradle check` in the terminal. Once this is done, you can start the application using the command `gradle bootrun`. If everything goes fine, the application will have been compiled and started Spring Framework with the application. Now, you can access from your favourite browser by typing in the address bar: *localhost:8080*.
+To build the project is required to download it or clone with git (`git clone https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race`). Now, you can see the code and this information. This code is built with gradle. Gradle is a build automation system which uses configuration files like build.gradle which is a build configuration script. You just have to follow the wiki of the main repository to configure the tools needed.
```
Comment (RecuencoJones) [inline description.md]: - `To build the project, first download it or clone...`
- `...which uses build configuration scripts like build.gradle.`

File: description.md [hunk 4]
```diff
@@ -47,22 +47,26 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
...
+
+After configuring Gradle you can try to compile using the command `gradle check` in the terminal, this will compiles and tests your code. Other command you can use to build the code is `gradle build` on your terminal. Gradle will compile and check your code and create a JAR file containing your main classes and resources.
```
Comment (RecuencoJones) [inline description.md]: `This will compile and test`*

File: description.md [hunk 5]
```diff
@@ -47,22 +47,26 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
...
+To build the project is required to download it or clone with git (`git clone https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race`). Now, you can see the code and this information. This code is built with gradle. Gradle is a build automation system which uses configuration files like build.gradle which is a build configuration script. You just have to follow the wiki of the main repository to configure the tools needed.
+
+After configuring Gradle you can try to compile using the command `gradle check` in the terminal, this will compiles and tests your code. Other command you can use to build the code is `gradle build` on your terminal. Gradle will compile and check your code and create a JAR file containing your main classes and resources. 
+
+Other possibilities related to `gradle build` are `gradle assemble` that compiles and jars your code, but does not run the unit tests. After a few seconds, gradle respond you with "BUILD SUCCESSFUL" that means the build has completed and you are able to run the application.
```
Comment (RecuencoJones) [inline description.md]: - You mention other possibilities but just mention one (assemble), either replace `are` for `like` or remove the plural or keep it single.
- `gradle will respond with`*

**Comment by RecuencoJones** [comment]
Looks good to me

LGTM

**Comment by fjlopez** [comment]
LGTM

**Pull request merged**